src/vscode-bicep/src/extensionVariables.ts

```diff
@@ -0,0 +1,16 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+import {
+  AzExtTreeDataProvider,
+  IAzExtOutputChannel,
+} from "@microsoft/vscode-azext-utils";
+
+class ExtensionVariables {
```

Comment:
We should avoid creating global variables and passing them around. See my comments https://github.com/Azure/bicep/pull/6129/files#r824086474 and https://github.com/Azure/bicep/pull/6129/files#r824103131 for refactoring suggestions.

Reply:
Removed it as part of this commit:
https://github.com/Azure/bicep/commit/62ec0b02cb7e0970c823c0b3f142ebe7fbc9b005

Thanks!